src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/policy/Conditional.kt

```diff
@@ -2,4 +2,6 @@ package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.policy
 
 data class Conditional(
   val inputs: List<ConditionalInput>,
-)
+) : HasInputs {
+  override fun getConditionInputs() = inputs.map { it.toInput() }
```

Comment:
Support a level of polymorphism to support using same mechanism to format conditional inputs as normal inputs